app/matplotlib.ipynb

```diff
@@ -159,25 +206,39 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
-    "\n",
-    "# write your code here\n"
+    "w = 1\n",
+    "plt.hist(data,rwidth=0.7)\n",
```

Comment:
The task requires making the bins size not so wide. Consider specifying the number of bins in the `plt.hist()` function to achieve this.